sup3r/preprocessing/data_handling/dual_data_handling.py

```diff
@@ -253,18 +251,15 @@ def normalize(self, means=None, stds=None, max_workers=None):
         super().normalize(means=means, stds=stds,
                           features=self.lr_dh.features,
                           max_workers=self.lr_dh.norm_workers)
+        self.lr_dh.normalize(means=means, stds=stds,
```

Comment:
Why do we need this? We should be able to compute stats without loading lr_dh.data.

Reply:
Ok but you're forcing it to be loaded a few lines above.

Reply:
I just fixed that exception raise clause in the last commit (i think this is what you're talking about and i agree this was leftover from previous)

Reply:
Ok nice. Works for me!